esque/cli/commands.py

```diff
@@ -232,7 +233,22 @@ def describe_topic(state, topic_name):
         topic = state.cluster.topic_controller.get_cluster_topic(topic_name)
         config = {"Config": topic.config}
 
+        high_watermark = 0
```

Comment:
Maybe move the `max_offset` functionality into it's own method and put it in the appropriate place in topic.py. Makes it also easier to test this.